tests/test.rs

```diff
@@ -18,7 +29,7 @@ fn create_socketpair() -> (Socket, Socket) {
     receiver.set_rcvtimeo(1000).unwrap();
 
     receiver.bind("tcp://*:*").unwrap();
```

Comment:
Yes, "127.0.0.1:*" would be just as effective/simpler; but once I dug into libzmq to verify the behavior was underlying and saw they had the same approach I figured it "made sense" here too, if only in that it provides some documentation of this behavior on windows. Although it may be worth adding such documentation to get_last_endpoint() itself explicitly?